### PR TITLE
DE45362 - Rubric popout read-only when user doesn't have correct permissions

### PR DESCRIPTION
**Pull request by tdevos-d2l**
[DE45362](https://rally1.rallydev.com/#/?detail=/defect/609981079925&fdp=true)
LMS PR: [https://github.com/Brightspace/lms/pull/14532](https://github.com/Brightspace/lms/pull/14532)
With new changes in the backend, users can now see the popout rubric without the 'Give Feedback' permission.
This change now makes sure that if they don't have that permission, they cannot edit the popout rubric. Done exactly the same as the right-panel rubric, with the `read-only` attribute.

### CODE REVIEW

**Review by JamieTran — APPROVED** [review]
LGTM

File: components/right-panel/consistent-evaluation-popup-rubric.js [hunk 1]
```diff
@@ -79,6 +81,14 @@ export class ConsistentEvaluationPopupRubric extends LocalizeConsistentEvaluatio
 			this.setTitle();
 
 			const evaluationEntity = await controller.getEvaluationEntity();
+
+			let hasWritePermission = false;
```
Comment (maddiegabriel) [inline components/right-panel/consistent-evaluation-popup-rubric.js:85]: should this be called `hasGiveFeedbackPermission`?

Comment (tdevos-d2l) [reply components/right-panel/consistent-evaluation-popup-rubric.js:85]: Sure, I'll change that.

**Review by maddiegabriel — APPROVED** [review]
👍

**Review by maddiegabriel — APPROVED** [review]
👍

**Pull request merged**